packages/editor-ui/src/views/SettingsApiView.vue

```diff
@@ -185,27 +137,22 @@ function onCopy() {
 			@click:button="onUpgrade"
 		/>
 		<n8n-action-box
-			v-else-if="mounted && !cloudPlanStore.state.loadingPlan"
+			v-if="isPublicApiEnabled && !apiKeysSortByCreationDate.length"
 			:button-text="
 				i18n.baseText(loading ? 'settings.api.create.button.loading' : 'settings.api.create.button')
 			"
 			:description="i18n.baseText('settings.api.create.description')"
-			@click:button="createApiKey"
+			@click:button="onCreateApiKey"
 		/>
 	</div>
 </template>
 
 <style lang="scss" module>
-.container {
-	> * {
-		margin-bottom: var(--spacing-2xl);
-	}
-}
-
 .header {
 	display: flex;
 	align-items: center;
 	white-space: nowrap;
+	margin-bottom: var(--spacing-2xl);
```

Comment:
we can add this as a class `.mb-2xl` now

Reply:
Since this is not the only thing that I'm applying I guess is better to group everything in a single class.